apps/jobs/serializers.py

```diff
@@ -27,12 +28,21 @@ def __init__(self, *args, **kwargs):
 
         super(SubmissionSerializer, self).__init__(*args, **kwargs)
 
+    def to_representation(self, obj):
```

Comment:
`baseline_submission` field is only for Challenge Hosts
>Add a column in the `My submissions`  table for challenge hosts only, so that they can toggle this functionality.
https://github.com/Cloud-CV/EvalAI/issues/1995

Reply:
yeah, but for participants the default is False and they can't toggle it using the API. Do we still need this in that case?

Reply:
@vkartik97 any updates on this?